jwst/combine_1d/tests/test_combine1d.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from numpy.testing import assert_allclose
+
+from ... master_background import create_master_bkg
+from ...combine_1d import combine1d
+
+
+def test_to_pixel():
+    wave = np.linspace(.3, 6.8, 10)
+    flux = np.arange(10, dtype=np.float64) + 100
+    bkg = create_master_bkg.create_background(wave, flux)
+    inp = combine1d.InputSpectrumModel(bkg, bkg.spec[0],
+                                       exptime_key='unit_weight',
+                                       background=False)
+    outm = combine1d.OutputSpectrumModel()
+    outm.assign_wavelengths([inp])
+    wave = [.3, 6.8, 2.2, .1]
+    res_to_pixel = outm.to_pixel(wave)
+    res_wcs = outm.wcs.invert(wave, wave, wave)
+    assert_allclose(res_to_pixel, res_wcs)
```

Comment:
The `to_pixel` test compares the results of `OutputSpectrumModel.to_pixel()` with `OutputSpectrumModel.wcs.invert()`. How should inputs outside the wavelength range be handled? Right now, `wcs.invert` will return `np.nan` for those while `to_pixel` extrapolates.

Reply:
I set the model in spec_wcs to allow extrapolation and now the test comparison with `to_pixel` passes.